interface/client/styles/layout.import.less

```diff
@@ -118,16 +105,6 @@ main {
     left: @widthSideBar;
     top: 0; 
     bottom: 0;
-
-    &.open-bar {
```

Comment:
@frozeman now in Mist repo at least 01 review is required in order to merge. if you approve the PR I can merge it right away.